api-lib/config.ts

```diff
@@ -61,6 +61,8 @@ export const HARDHAT_GANACHE_PORT: number = getEnvValue(
 
 export const ETHEREUM_RPC_URL: string = getEnvValue('ETHEREUM_RPC_URL');
 
+export const GOERLI_RPC_URL: string = getEnvValue('GOERLI_RPC_URL');
```

Comment:
We should add this to the `.env.example` as it was failing to initialise without it, created the PR https://github.com/coordinape/coordinape/pull/1055 to add it